### ISSUE TITLE
Why my loss of dev increases steadily？？？？

### ISSUE THREAD

**Issue by xiayanan** [dennybritz/cnn-text-classification-tf#56]

**Comment by patfol** [comment]
I have the same issue, with default parameter and exact same code as in tutorial my dev loss increases steadily, even when performing 2000 epochs, while dev accuracy stays stable and train loss has reached 0.

I thought maybe it was a bug, but with mnist tutorial also using cross entropy loss the dev loss reached a plateau. 

I have tried different loss, and with applying a "dropout to labels" as in https://arxiv.org/pdf/1512.00567.pdf, with label smoothing of 0.1 my dev loss became stable.

self.loss = tf.losses.softmax_cross_entropy(self.input_y, self.scores, label_smoothing=0.1)    

I'm a begginer in ML if someone has an idea of what is happening I'd be gratefull!

![lossproblem](https://cloud.githubusercontent.com/assets/24589803/23896522/07dc4b4e-08aa-11e7-95bd-8f44e40f0742.jpg)

**Comment by JulianFuchs** [comment]
I think the paper is not very well explained in that point. As far as I understand it, the increase in dev loss is explained with overfitting the training data.

I first thought that I had made a mistake since the tutorial says that the figures showing loss and accuracy are achieved with the default parameters. But that's just false in my opinion.

It makes sense that without regularization the model overfits the training data, so the loss of the training set gets 0 but the loss of the test set increases since the model chooses higher parameters fitting the training data perfectly, but increasing the loss for all other data. I ran the model again with a regularization of 3 and the loss does not increase anymore...

On a completely unrelated note, the first picture  showing the model presented in the tutorial was quite confusing. The picture seems to be taken from a paper which uses word embeddings, but the tutorial doesn't use word embeddings. Took me quite a while to figure that one out....
